.travis.yml

```diff
@@ -13,12 +13,15 @@ notifications:
 script:
   bundle exec rspec spec && bundle exec rubocop
 
-deploy: 
+after_success:
+  ./scripts/sync_repos.sh
+
+deploy:
   provider: elasticbeanstalk
   app: lvm-rails-production
   env: Sample-env
   bucket_name: elasticbeanstalk-us-east-1-373469085898
   region: us-east-1
   on:
     repo: LiteracyVolunteersOfMA/lvm-rails
-    all_branches: true  
```

Comment:
This part of the travis file is specifying deployment information telling travis when to deploy to AWS. There's really no good way to know if we're in a pull request and/or a master branch build without checking environment variables.